reaktive/src/jvmCommonMain/kotlin/com/badoo/reaktive/utils/PrioritySerializer.kt

```diff
@@ -0,0 +1,59 @@
+package com.badoo.reaktive.utils
+
+import java.util.*
```

Comment:
Don't use `*` imports. Disable this setting in IDEA Kotlin Code Style settings.

Reply:
Fixed. Perhaps we should commit IDEA settings to the repo.

Reply:
And then have problems with `.codestyle` file on different versions of IDEA and Android Studio?)